docker/Dockerfile

```diff
@@ -31,6 +31,12 @@ RUN if [ -n "${INSTALL_CERT}" ]; then \
     -O /usr/local/share/ca-certificates/custom.crt \
     && update-ca-certificates; fi
 
+ARG ROOT_CERT=
```

Comment:
@glattercj I'm wondering if we shouldn't just check for a comma-separated list of cert URLs for `INSTALL_CERT`? That would make it future-proof. Otherwise, can we use `custom-root.crt` instead of `root.crt` for this new certificate?

Reply:
Yes, definitely. I was just being lazy...

Reply:
Sheesh... like you have something better to be doing. :joy:

Reply:
If you can make it prettier please do.